src/pages/components/progress-bar/style.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Progress bar
+description:
+  A progress bar provides feedback about the duration and progression of a
+  process.
+tabs: ['Usage', 'Style']
+---
+
+## Color
+
+| Element       | Property   | Color token           |
```

Comment:
Just for consistency we usually do not capitalize the text in the rows under the "Property" columns.